t/ui/10-tests_overview.t

```diff
@@ -565,23 +581,25 @@ subtest 'add comments' => sub {
       'comments created on all relevant jobs';
     is $comments->search({job_id => {-not_in => \@failed_job_ids}, text => $comment_text})->count, 0,
       'comments not created on other jobs';
-};
-
-subtest "job template names displayed on 'Test result overview' page" => sub {
-    $driver->get('/group_overview/1002');
-    is($driver->find_element('.progress-bar-failed')->get_text(), '1 failed', 'The number of failed jobs is right');
-    is($driver->find_element('.progress-bar-unfinished')->get_text(),
-        '1 unfinished', 'The number of unfinished jobs is right');
-
-    $driver->get('/tests/overview?distri=opensuse&version=13.1&build=0091&groupid=1002');
-    my @tds = $driver->find_elements('#results_DVD tbody tr .name');
-    is($tds[0]->get_text(), 'kde_variant', 'job template name kde_variant displayed correctly');
 
-    my @descriptions = $driver->find_elements('td.name a', 'css');
-    is(scalar @descriptions, 2, 'only test suites with description content are shown as links');
-    disable_bootstrap_animations;
-    $descriptions[0]->click();
-    is(wait_for_element(selector => '.popover-header')->get_text, 'kde_variant', 'description popover shows content');
+    subtest 'restart jobs with comment' => sub {
+        $driver->get('/tests/overview?state=done&result=failed');
+        disable_bootstrap_animations;
```

Comment:
I don't think we need this function anymore. So don't add it in new code. You might also remove it in the other test that is affected by the diff of this PR. (If I turn out to be wrong and this is still needed disregard this comment.)